app/public/templates/scene.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>{{ scenario_dict["name"] }} - {{ scene_dict["name"] }}</title>
+    <title>{{ scenario_name }} - {{ scene_dict["name"] }}</title>
```

Comment:
I can't add a comment on line 25-26, but this is for that:
Since we are adding the `is_admin` parameter, can we use that for that line similar to camera (https://github.com/uwblueprint/cdc-backend/blob/master/app/public/templates/scene.html#L78) for adding inspector:
```
<a-scene id="{{ scene_dict['id'] }}-scene" cursor="rayOrigin:mouse" raycaster="objects: .link"
{% if is_admin %}
            inspector="url: {{ inspector_url }}"
 {% end %}
>
```
And then in the python admin scenario handler:
```
inspector_url=config.get("inspector_url")
```
And add this one to dev-config:
inspector_url: http://localhost:3333/dist/aframe-inspector.js

That way, when switching to prod, we just have to update the config and the rest will work out.

Reply:
🧠